### DISCUSSION TITLE
Symfony 5.4 format widget to display year only

### DISCUSSION THREAD

**Discussion by vlaup** [symfony/symfony#49110] [Q&A]
The current buildForm with the year added
`->add('year', DateType::class, array(`
                `'label' => 'Select Year',`
                `'widget' => 'choice',`
               `'years' => range(Date('Y'), 2000),`

twig file to customize the month / day attributes, this used to work

`{{ form_widget(form.year['day'], {'attr':{'style':'display:none'}}) }}`
`{{ form_widget(form.year['month'], {'attr':{'style':'display:none'}}) }}`

It currently displays this, would like it to only display the year
![image](https://user-images.githubusercontent.com/123605622/214724188-61df3b70-0a52-4659-a364-b8a3922c39c7.png)

**Comment by MatTheCat** [comment]
Maybe I’m missing something but if you just need a year the `IntegerType` would be more appropriate.
